src/filebrowser/file-browser-dialog.cpp

```diff
@@ -640,9 +640,9 @@ void FileBrowserDialog::uploadMultipleFile(const QStringList& names,
     QStringList fnames;
     Q_FOREACH(const QString &name, names) {
         const QFileInfo file = name;
-        // only files are allowed
-        if (!file.isDir() && file.path() == path
-            /* && !account_.isPro*/) {
+        if (file.isDir()) {
```

Comment:
Sorry, I don't know if this is for me. Unfortunately I don't understand Chinese / Mandarin.